Analyzer/src/AnalyzeDoubleDisCo.cc

```diff
@@ -1623,8 +1645,12 @@ void AnalyzeDoubleDisCo::Loop(NTupleReader& tr, double, int maxevents, bool isQu
                                     //  -- each of A, B, C, D has 7,8,9,10,11, 12incl Njets bins
                                     // ---------------------------------------------------------
                                     if (njets == "Incl")
-                                        my_histos["h_njets_12incl" + name]->Fill(theGoodJets>=12 ? 12-theMVAjets+shift*6 : theGoodJets-theMVAjets+shift*6, w);
-
+                                    {
+                                        //theMVAjets This is basically hardcoded to be 7 always. However, in the future it shouldn't and the following lines need to be updated then
+                                        my_histos["h_njets_11incl" + name]->Fill(theGoodJets>=11 ? 11 - (theMVAjets-1) + shift*6 : theGoodJets - (theMVAjets-1) + shift*6, w);
```

Comment:
In any case, irrespective of my immediately above comment, the logic here should work given that `theMVAjets` always comes out to 7.